components/lvgl_esp32_drivers/lvgl_tft/sh1107.c

```diff
@@ -165,14 +161,25 @@ void sh1107_flush(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * colo
 #else 
         ptr = color_map + i * CONFIG_LVGL_DISPLAY_WIDTH;
 #endif
-        sh1107_send_color( (void *) ptr, size);
+        if(i != row2){
+			sh1107_send_data( (void *) ptr, size);
+		} else {
+			// complete sending data by sh1107_send_color() and thus call lv_flush_ready()
+			sh1107_send_color( (void *) ptr, size);
+		}
```

Comment:
I'm generally not picky about indentation but this is confusing.

Reply:
I will fix that. Didn't noticed it.